org.devloop.Do.json

```diff
@@ -0,0 +1,39 @@
+{
+    "app-id" : "org.devloop.Do",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-nightly"
+    ],
+    "command" : "todo",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-nightly/bin",
+        "build-args" : [
+            "--share=network"
+        ],
```

Comment:
this is not allow and will fail
you need to add the cargo dependencies in the manifest.

See https://github.com/flatpak/flatpak-builder-tools

Optionaly a vendored tarball instead if git works if you use standard Rust meson templates.